doc-content/enterprise-only/fuse/kie-camel-con.adoc

```diff
@@ -0,0 +1,60 @@
+[id='kie-camel-con_{context}']
+= The `kie-camel` component
+
+The `kie-camel` component is an Apache Camel endpoint provided by {FUSE_LONG} that integrates {FUSE} with {PRODUCT}.  It enables you to specify a {PRODUCT} module by using a Maven group ID, artifact, and version (GAV)  which you can pull into the route and execute. It also enables you to specify portions of the message body as facts.You can use the `kie-camel` component with embedded engines or with {KIE_SERVER}.
```

Comment:
"artifact ID, and version (GAV) identifier that you"  Main thing is that it's "artifact ID". See my previous GAV comment.

"facts. You" (space) Though I suggest a line break, so it's a clear lead-in to the two segments that follow.